packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -144,7 +144,7 @@ void main() {
     expect(find.byType(ExpandIcon), findsOneWidget);
     await tester.tap(find.byType(ExpandIcon));
     expect(capturedIndex, 0);
-    expect(capturedIsExpanded, isFalse);
+    expect(capturedIsExpanded, isTrue);
```

Comment:
Since this issue has existed for a while, developers may be broken by this. I am going to run some extra tests internally to see if it breaks anyone.

Reply:
Ok sounds good, thanks again for the help.